builder/config_reader_test.go

```diff
@@ -84,11 +84,10 @@ func testConfig(t *testing.T, when spec.G, it spec.S) {
 `), 0666))
 				})
 
-				it("returns warnings", func() {
+				it("returns error when obsolete 'groups' field is used", func() {
```

Comment:
Technically with this change we could remove this function call (and its definition): https://github.com/buildpacks/pack/pull/535/files#diff-9e3eaa38298a8257cec0740ad5e24fd9R79 - but we can also save this for #538.

Reply:
I am a bit unsure about this. If we just remove the function call then its fine and we can address #538 with this too. But if we save this for issue #538, then what would we leave here? Means after the buildpack validations, we can either remove the test or identify it as an error in the test. 
@natalieparellano

Reply:
I guess it would be better if this function is removed for now. Your opinions?